nilearn/plotting/html_surface.py

```diff
@@ -297,9 +297,10 @@ def view_surf(surf_mesh, surf_map=None, bg_map=None, threshold=None,
     if surf_map is None:
         surf_map = np.ones(len(surf_mesh[0]))
     if surf_map is not None:
```

Comment:
SHouldn't we just make this `else:`?

Reply:
:laughing: why make things simple when you can make them complicated?